frontend/src/pages/HomePage.tsx

```diff
@@ -1,17 +1,165 @@
-import './../App.css';
 import LogoutButton from "../components/LogoutButton";
-export default function HomePage () {
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import {toast} from "react-toastify";
+import "material-react-toastify/dist/ReactToastify.css";
+import {Item} from "../model/Item";
+import DeleteIcon from '@mui/icons-material/Delete';
 
+import {
+    AppBar,
+    Box,
+    Button,
+    Card, CardActions, CardContent, CardMedia,
+    createTheme,
+    CssBaseline,
+    Grid,
+    Stack,
+    ThemeProvider,
+    Typography
+} from "@mui/material";
+import Container from "@mui/material/Container";
+import Toolbar from "@mui/material/Toolbar";
+import {useNavigate} from "react-router-dom";
 
+export default function HomePage() {
+    const [items, setItems] = useState([] as Item[]);
 
-  return (
-    <div className="App">
-      <h1>PLATZHALTER HOMEPAGE</h1>
-        <br></br>
-        <h1> WIRD NOCH SCHÖNER!</h1>
-      <br/>
-      <br/>
-      <LogoutButton/>
-    </div>
-  );
+    useEffect(() => {
+        (async () => {
+            axios.get("/api/items")
+                .then((response) => setItems(response.data))
+                .catch((error) => toast.error(error.message));
+        })();
+    }, []);
+
```

Comment:
Isn't this useEffect the same as your loadItems function?

Reply:
Ja, stimmt. Ich habe den Button noch als Hilfsmittel drin, nehme ich aber jetzt raus.